src/fusion_power_plant.cc

```diff
@@ -193,12 +199,16 @@ bool FusionPowerPlant::TritiumStorageClean() {
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 bool FusionPowerPlant::ReadyToOperate() {
 
-  // determine required tritium storage inventory
-  double required_storage_inventory = fuel_usage_mass + SequesteredTritiumGap();
+  // Determine tritium inventory required to operate
+  if (sequestered_tritium->quantity() < cyclus::eps_rsrc()) {
+    double required_storage_inventory = reserve_inventory + SequesteredTritiumGap();
+  } else {
+    double required_storage_inventory = fuel_usage_mass + SequesteredTritiumGap();
+  }
```

Comment:
Because of limitations of C++ scope this may need to declare the variable outside of the if statement:
```suggestion
  // Determine tritium inventory required to operate
  double required_storage_inventory = SequesteredTritiumGap();
  if (sequestered_tritium->quantity() < cyclus::eps_rsrc()) {
    required_storage_inventory += reserve_inventory;
  } else {
    required_storage_inventory += fuel_usage_mass;
  }
```

Reply:
I think this has been fixed.